code/game/objects/items/holy_weapons.dm

```diff
@@ -390,8 +391,31 @@
 	lefthand_file = 'icons/mob/inhands/weapons/polearms_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/polearms_righthand.dmi'
 	name = "unholy pitchfork"
-	w_class = WEIGHT_CLASS_NORMAL
+	w_class = WEIGHT_CLASS_BULKY
 	desc = "Holding this makes you look absolutely devilish."
 	attack_verb = list("poked", "impaled", "pierced", "jabbed")
+	slot_flags = SLOT_BACK
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	sharpness = IS_SHARP
+	force = 10
+	bonus_burn = 5
+	
+/obj/item/nullrod/pitchfork/clockworkspear/clockwork 
+	icon = 'icons/obj/clockwork_objects.dmi'
+	icon_state = 'ratvarian_spear'
```

Comment:
Using single quotes like this indicated you want BYOND to find a file that matches, not a string. Use `" "` instead.